### PR TITLE
Add Dapr icon

### PR DESCRIPTION
**Pull request by helbing**
dapr

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

- Dapr is a GitHub project with [over 19.2k stars](https://github.com/dapr/dapr)
- I picked the color from https://github.com/dapr/dapr/blob/master/img/dapr_logo.svg

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2636,6 +2636,11 @@
             "source": "https://designnavigator.daimler.com/Daimler_Corporate_Logotype_Black_DTP",
             "guidelines": "https://designnavigator.daimler.com/Daimler_Corporate_Logotype"
         },
+        {
+            "title": "Dapr",
+            "hex": "0D2192",
+            "source": "https://github.com/dapr/dapr/tree/master/img"
```
Comment (LitoMore) [inline _data/simple-icons.json:2642]:
```suggestion
            "source": "https://raw.githubusercontent.com/dapr/dapr/f5a5acc406302f0d5122ae30d18f9baba6dba8d3/img/dapr_logo.svg"
```

Comment (LitoMore) [reply _data/simple-icons.json:2642]: Let's use permalink instead.

**Review by LitoMore — CHANGES_REQUESTED** [review]

**Comment by LitoMore** [comment]
CleanShot 2022-09-08 at 14 33 36@2x

@helbing I see some differences, could you give a check on this?

File: _data/simple-icons.json (hunk 1 above)
Comment (helbing) [reply _data/simple-icons.json:2642]: > Let's use permalink instead.

Okay 👌

Comment (LitoMore) [reply _data/simple-icons.json:2642]: > How do I reproduce this problem? (@helbing)

You could refer to https://github.com/simple-icons/simple-icons/pull/7752#issuecomment-1238557522.

Comment (helbing) [reply _data/simple-icons.json:2642]: > 

thk

**Comment by LitoMore** [comment]
CleanShot 2022-09-08 at 16 37 18@2x

@helbing Still not fit.

The expected aspect ratio should be:

CleanShot 2022-09-08 at 16 38 02@2x

But yours:

CleanShot 2022-09-08 at 16 38 59@2x

Some SVG editors might snap to pixels by default. Please make sure you didn't enable that kind of feature on your editor.

**Review by LitoMore — CHANGES_REQUESTED** [review]

**Comment by helbing** [comment]
> CleanShot 2022-09-08 at 16 37 18@2x
> 
> @helbing Still not fit.
> 
> The expected aspect ratio should be:
> 
> CleanShot 2022-09-08 at 16 38 02@2x
> 
> But yours:
> 
> CleanShot 2022-09-08 at 16 38 59@2x
> 
> Some SVG editors might snap to pixels by default. Please make sure you didn't enable that kind of feature on your editor.

My mistake, It's done now. w: 24, h: 17.636, x: 0, y: 3.182

iShot_2022-09-08_17 34 04

**Review by LitoMore — APPROVED** [review]
CleanShot 2022-09-08 at 22 58 16@2x

@helbing Perfect fit! Thank you for your contributions!

**Pull request merged**